packages/components/src/components/remaining-time/remaining-time.tsx

```diff
@@ -1,9 +1,16 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { formatDuration, getDiffDuration } from '@deriv/shared';
+import moment from 'moment';
 
-const RemainingTime = ({ end_time = null, format, getCardLabels, start_time }) => {
-    if (!+end_time || start_time.unix() > +end_time) {
+type TRemainingTimeProps = {
+    end_time?: number | null;
```

Comment:
no need to declare type as `null` since the field is already nullable

Reply:
@jim-deriv  I also can't find this prop to be passed from outside as null. Is there such a place? 🙂

Reply:
Alright guys. I'll remove null from the prop :sweat_smile:

cc: @yashim-deriv, @maryia-binary @farzin-fs